P001/modelsim.mk

```diff
@@ -0,0 +1,22 @@
+
+all: work
+	vcom -work work P001.vhdl
```

Comment:
-work work is optional.

You can also use vcom -check_synthesis P001.vhdl, but all it checks really is sensitivity list.